app/frontend/src/components/objectList/ShareObject.vue

```diff
@@ -26,24 +26,41 @@
         <!-- Invite a specific user -->
         <div v-if="shareMode === 'user'">
           <h2>Invite</h2>
-          <v-row>
+
+          <ul class="mt-2">
+            <li>This will grant the READ permission on this file to another user</li>
```

Comment:
Depends on who we are targeting as a demo/users. If we're abstracting it out a bit, we probably don't need to be "precise" about what permission is being granted, and instead say that "This will allow another user to read this file".